README.md

```diff
@@ -129,7 +78,7 @@ Finally, run the replicator:
 poetry run cdf_fabric_replicator config.yaml
 ```
 
-## Visual Studio Code
+### Visual Studio Code
```

Comment:
Is this section needed?

Reply:
If we're going to have this section might be worth explaining the launch configs that are available (poetry install, poetry run)

Reply:
I do think it's worth having the section. I broke it down as Maya suggested to provide better explanations of the launch configs.